dascore/io/sintela_protobuf/utils.py

```
@@ -0,0 +1,1065 @@
+    return ParsedMeta(
+        recorder_namespace=str(getattr(msg, "recorder_namespace", "") or ""),
+        metadata_recording_time=(
+            _timestamp_to_dt64(msg.metadata_recording_time)
+            if msg.HasField("metadata_recording_time")
+            else None
+        ),
+        instrument_manufacturer=str(getattr(identification, "manufacturer", "") or ""),
+        instrument_model=str(getattr(identification, "model", "") or ""),
+        serial_number=str(getattr(identification, "serial_number", "") or ""),
+        fiber_id=(
+            int(getattr(acquisition, "fiber_id"))
```

Ruff (0.15.9)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)